detour.go

```diff
@@ -124,6 +127,9 @@ func SetCountry(country string) {
 // Dialer returns a function with same signature of net.Dialer.Dial().
 func Dialer(d dialFunc) dialFunc {
 	return func(network, addr string) (conn net.Conn, err error) {
+		if allowsDirect(addr) {
+			return netx.DialTimeout(network, addr, TimeoutToDetour)
```

Comment:
I would think we'd want this to be closer to one of our more typical dial timeouts instead of 3 seconds @oxtoacart